src/reaktoro_pse/reaktoro_block.py

```diff
@@ -0,0 +1,952 @@
+                - average methods takes defined number of derivatives by numerical_jac_order from center points and gets the average of them
+                - center_difference methods applies classical taylor difference approximation methods 
+            In theory the two should yield same result- but due to round off errors the average method might provide better error dampening. 
+
+            """,
+        ),
+    )
+    CONFIG.declare(
+        "numerical_jac_order",
+        ConfigValue(
+            default=8,
+            domain=int,
+            description="Defines order of numerical jacobian",
+            doc="""
+            This will define how many points to discretize the derivate over 
+             - for numerical_jac_type==average - the number of points = order/2 + 1 
+             - for numerical_jac_type==center_difference - the number of points equals the order
+            """,
+        ),
+    )
+    CONFIG.declare(
+        "numerical_jac_step",
+        ConfigValue(
+            default=1e-6,
+            domain=float,
+            description="Defines the step to use for numerical descritiazaiton",
+            doc="""This will define how small of a step to use for numerical derivative propagation which takes
+             the absolute chemical property and multiplies it by chemical property derivative multiplied by step 
+                chemical_property_step=chemical_property_absolute_value*chemical_property_derivative*step
+            """,
+        ),
+    )
+    CONFIG.declare(
+        "jacobian_scaling_type",
+        ConfigValue(
+            default=JacScalingTypes.variable_scaling,
+            domain=IsInstance((str, JacScalingTypes)),
+            description="Defines how to scale Jacobian matrix",
+            doc="""
+            Defines methods for jacobian scaling:
+            - if option is no_scaling, jacobian scale will == 1 for all outputs
+            - if option is 'variable_scaling' will use output variable scaling factors
+            - if option is jacobian_matrix will use actual jac matrix to calculate scaling factors
+            - if user_scaling is not None then uses user provided scaling
+            """,
+        ),
+    )
+    CONFIG.declare(
+        "jacobian_user_scaling",
+        ConfigValue(
+            default=None,
+            domain=IsInstance((float, list, dict)),
+            description="Manual scaling factors for jacobian",
+            doc="""
+            Applies user provided jacobian scaling values:
+             - either single value that will be applied to all outputs in jacobian
+             - array applied across jacobian
+             - dict that specifics output and scaling factor to which apply scaling, (variable_scaling will be applied to non specified outputs)
+                e.g. {output_name:scaling_factor} applies to specific jac output 
+            """,
+        ),
+    )
+    CONFIG.declare(
+        "tolerance",
```

Comment:
Updated to rename to reaktoro_solver_tolerance

Reply:
This might be another place to consider nested ConfigDicts - it might be worth having a `reaktoro_config` sub-dict that holds any arguments that go straight to reaktoro (and thus avoid the need to put `reaktoro_` in front of all of them).